test/BaseTest.sol

```diff
@@ -67,7 +69,10 @@ contract BaseTest is Test {
     address userProxyAddr;
     COWShed userProxy;
     COWShed cowshedImpl = new COWShed();
-    COWShedFactory factory = new COWShedFactory(address(cowshedImpl));
+    bytes32 baseName = "cowhooks.eth";
+    bytes32 baseNode = 0x8f8ff4ebd0ab36d517ed7387211b245b7d5508f4ed33be607a3ab54050f650d6;
```

Comment:
what is this special value?

Reply:
its the namehash of cowhooks.eth, changed to call the cheatcode instead